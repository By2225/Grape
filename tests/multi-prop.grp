fun Int main() {

        List<Node<Int> > a = ['3', '4'];
        
        print(a[0].val);
        return a[1].val;
}
