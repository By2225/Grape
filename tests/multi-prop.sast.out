
fun Int main() {
List<Node<Int>> (List<Node<Int>> : a = (List<Node<Int>> : [(Node<Int> : '(Int : 3)'),(Node<Int> : '(Int : 4)')]));
(* : print((Int : (Node<Int> : (List<Node<Int>> : a)[(Int : 0)]).val)));
(* : print((Int : (Node<Int> : '(Int : 3)').val)));
return (Int : 0);
}
